tests/mr/device/cuda_async_mr_tests.cpp

```diff
@@ -66,5 +66,30 @@ TEST_F(AsyncMRTest, DifferentPoolsUnequal)
   EXPECT_FALSE(mr1.is_equal(mr2));
 }
 
+TEST_F(AsyncMRTest, FabricHandlesSupport)
+{
+  const auto pool_init_size{100};
+  const auto pool_release_threshold{1000};
+  cuda_async_mr mr{pool_init_size,
```

Comment:
oh, I am going to need to feature flip these tests. Calling the same RMM function to detect whether fabric is supported.